go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/honeycombio/libhoney-go
 
 go 1.14
 
+replace (
+	github.com/vmihailenco/msgpack/v4 => gopkg.in/vmihailenco/msgpack.v4 v4.3.11
+	gopkg.in/vmihailenco/msgpack.v4 => github.com/vmihailenco/msgpack/v4 v4.3.11
```

Comment:
Doesn't this create a circular replacement or do I misunderstand?

Reply:
Sorry, I should have added a comment or at least a note on the PR, because this is indeed confusing.

The `gopkg.in` line is needed to replace the import paths in `libhoney-go`. The `github.com` line is needed to replace import paths in `msgpack` itself. Astoundingly, `go get` manages to resolve all of this and do the right thing! It took me a few goes to figure this out. 😅

Would you like me to add a comment?

Reply:
yes please.

Reply:
Done!